topics/assembly/tutorials/general-introduction/tutorial.md

```diff
@@ -28,196 +32,306 @@ We will now import the data that we will use for the tutorial.
->
-> ![Contigs output](../../images/image10.png)
->
->   - The *Contigs stats* file will show a list of these k-mer lengths and k-mer coverages.
->
-> ![Contigs stats output](../../images/image11.png)
+    ![Sequence Quality Histograms with the mean quality value across each base position in the read](../../images/fastqc_per_base_sequence_quality_plot.png "The mean quality value across each base position in the read")
+
+    > ### {% icon question %} Questions
+    >
+    > 1. What does the y-axis represent?
+    > 2. Why is the quality score decreasing across the length of the reads?
+    >
+    >    <details>
+    >    <summary>Click to view answer</summary>
+    >    <ol type="1">
+    >    <li>The y-axis represents the quality score for each base (an estimate of the error during sequencing)</li>
+    >    <li>The quality score is decreasing accross the length of the reads because the sequencing become less and less reliable at the end of the reads</li>
+    >    </ol>
+    >    </details>
+    {: .question}
+
+- Per Sequence GC Content
+    High GC organisms don't tend to assemble well and may have an uneven read coverage distribution.
+- Per Base N Content
+    Presence of large numbers of Ns in reads: may point to poor quality sequencing run. You would need to trim these reads to remove Ns.
+- Kmer content
+    Presence of highly recurring k-mers: may point to contamination of reads with barcodes or adapter sequences.
+
+
+> ### {% icon comment %} Comment
 >
+> For a fuller discussion of FastQC outputs and warnings, see the [FastQC website link](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/), including the section on each of the output [reports](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/), and examples of ["good"](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/good_sequence_short_fastqc.html) and ["bad"](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/bad_sequence_fastqc.html) Illumina data.
+{: .comment}
+
+We won't be doing anything to these data to clean it up as there isn't much need. Therefore we will get on with the assembly!
+
+
+# Assemble reads with Velvet
+
+Now, we want to assemble our reads to find the sequence of our imaginary *Staphylococcus aureus* bacterium. We will perform a *de novo* assembly of the reads into long contiguous sequences using the Velvet short read assembler. 
+
+The first step of the assembler is to build a de Bruijn graph. For that, it will break our reads into k-mers, *i.e.* fragments of length *k*. Velvet requires the user to input a value of *k* (k-mer size) for the assembly process. Small k-mers will give greater connectivity, but large k-mers will give better specificity.
+
+> ### {% icon hands_on %} Hands-on: Assemble the reads
+>
+> 1. **FASTQ interlacer** {% icon tool %} with the following parameters
```

Comment:
Velvet can take separate files for read pairs. Why are we interlacing here?

Reply:
Oh, I see you're using the old velvet tool. With separate velveth and velvetg. And the velveth tool needs it. Is there a reason for using the old tool instead of the one I bundled with the optimiser? I usually combine the steps as all newer assemblers don't do these steps separately and I encourage people to never use Velvet outside of the tutorial!

Reply:
I agree to not encourage people to use Velvet outside of the tutorial. 
I used the velvet tool I found on Freiburg instance 😄 
But I found also pedagogic the idea of the 2 steps to explain how the assembly is done: first breaking your reads into k-mers and then doing the assembly *per se*. But if you think it is better to use your tool, I can change that 😄

Reply:
No. That's ok. I just wanted to know why. Maybe we should update that tool to use the -separate flag.. All good.